tests/stub/transport/test_handshakes.py

```diff
@@ -83,10 +85,61 @@ def test_handshake_manifest_server_responses(self):
                 "00 00 01 FF 02 00 07 07 F2 00 07 07 05 00",
                 "00 00 07 05 00",
             ),
-
         ]:
             with self.subTest(
                 server_response=server_response,
                 expected_response=expected_response
             ):
                 self._run(server_response, expected_response)
+
+    def _get_newest_bolt_supported_by_driver(self, skip=0, min_version=(5, 0)):
+        # skip: number of newest versions to skip (return n-th newest)
+        # min_version: minimum bolt version to consider
+        all_bolt_versions = [
+            (f, tuple(map(int, f.value.split(":")[-1].split("."))))
+            for f in types.Feature
+            if re.match(r"BOLT_(\d+_)*(\d+)", f.name)
```

Comment:
```suggestion
            if re.match(r"^BOLT_(\d+)_(\d+)$", f.name)
```
I feel like a somewhat less permissive regex expression can be used here. This prevents feature names like "EXPLOSIVE_BOLT_5_7" or "BOLT_5_7_KABOOM" from matching the regex.

Reply:
Yup. That regex is also used in other places in TestKit. I'll adjust them as well, while I'm at it :bow: